main-site/components/mailinglist-signup/MailinglistSignup.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import addToMailChimp from 'gatsby-plugin-mailchimp';
+import { StyledEmailInput } from '../../../shared-ui/components/footer/Footer.styles';
```

Comment:
If you're making a new MailingListSignup component then move anything that you're styling for this into MailingListSignup.styles.ts